views/default/simplesaml/settings/service_provider_advanced.php

```diff
@@ -50,6 +50,19 @@
 	$body .= elgg_format_element('div', [], $external_id);
 }
 
+$force_authentication = $plugin->getSetting('force_authentication');
+if ($force_authentication == $source_id) {
```

Comment:
This means you only see this feature after you save the settings first... not a very big issue